plugins/ltac2/tac2tactics.mli

```diff
@@ -118,6 +118,8 @@ val typeclasses_eauto : Class_tactics.search_strategy option -> int option ->
 
 val unify : constr -> constr -> unit tactic
 
+val evarconv_unify : constr -> constr -> unit tactic
```

Comment:
I'm not sure that exposing the internal name of the Evarconv module in the API is a good idea. I don't have any other proposal though.

Reply:
Yeah, I completely agree. And I obviously did not have any better idea either.

Reply:
`new_unify`? "New unfication" and "evarconv" are the two terms I hear the most.

Reply:
Evarconv seems the best established term honestly.

IIRC Hugo pointed out "new unification" isn't actually newer... current docs/option seem to use "unification" (evarconv) and "tactic unification", but `unify` seems to violate that convention, and "unification" seems best reserved for the generic concept that both evarconv and tactic unification refine.

Reply:
Going with a new module.